README.md

```diff
@@ -343,315 +375,340 @@ convert asff2hdf            Translate a AWS Security Finding Format JSON into a
 
 ##### Prisma to HDF
-
 ```
-convert prisma2hdf         Translate a Prisma Cloud Scan Report CSV file into a Heimdall Data Format
-                          JSON file
-  OPTIONS
-    -i, --input=input          Prisma Cloud Scan Report CSV
-    -o, --output=output        Output HDF JSON File
+convert prisma2hdf            Translate a Prisma Cloud Scan Report CSV file into
+                              Heimdall Data Format JSON files
+  USAGE
+    $ saf convert prisma2hdf -i <prisma-cloud-csv> -o <hdf-output-folder> [-h]
 
-  DESCRIPTION
-    Note: This mapper generates multiple output files, named by corresponding host names
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Prisma Cloud Scan Report CSV
+    -o, --output=<value>  (required) Output HDF JSON file
 
   EXAMPLES
-    saf convert prisma2hdf -i prisma.csv -o output-hdf-name.json
+    $ saf convert prisma2hdf -i prismacloud-report.csv -o output-hdf-name.json
 ```
 
 ##### Prowler to HDF
-
 ```
-convert prowler2hdf         Translate a Prowler-derived AWS Security Finding
-                            Format results from concatenated JSON blobs into a
-                            Heimdall Data Format JSON file
-  OPTIONS
-    -i, --input=input          Input Prowler ASFF JSON File
-    -o, --output=output        Output HDF folder
+convert prowler2hdf           Translate a Prowler-derived AWS Security Finding
+                              Format results from concatenated JSON blobs
+                              into a Heimdall Data Format JSON file
+  USAGE
+    $ saf convert prowler2hdf -i <prowler-finding-json> -o <hdf-output-folder> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Prowler ASFF JSON File
+    -o, --output=<value>  (required) Output HDF JSON Folder
 
   EXAMPLES
-    saf convert prowler2hdf -i prowler-asff.json -o output-hdf-name.json
+    $ saf convert prowler2hdf -i prowler-asff.json -o output-folder
 ```
 
-
 ##### Sarif to HDF
-
 ```
-convert sarif2hdf          Translate a SARIF JSON file into a Heimdall Data
-                            Format JSON file
-  OPTIONS
-    -i, --input=input          Input SARIF JSON File
-    -o, --output=output        Output HDF JSON File
+convert sarif2hdf             Translate a SARIF JSON file into a Heimdall Data
+                              Format JSON file
+  USAGE
+    $ saf convert sarif2hdf -i <sarif-json> -o <hdf-scan-results-json> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input SARIF JSON File
+    -o, --output=<value>  (required) Output HDF JSON File
 
   DESCRIPTION
     SARIF level to HDF impact Mapping:
-      SARIF level error -> HDF impact 0.7
-      SARIF level warning -> HDF impact 0.5
-      SARIF level note -> HDF impact 0.3
-      SARIF level none -> HDF impact 0.1
-      SARIF level not provided -> HDF impact 0.1 as default
+    SARIF level error -> HDF impact 0.7
```

Comment:
why is this information even provided in the readme?  we don't do it for any of the others.  can you ask @ejaronne what the historical usecase for putting this in the readme was?  otherwise let's just remove this section.

Reply:
Will do

Reply:
These mappings are apparently so novel, that they are worth mentioning in the description. So for now we'll keep them there, and maybe might even have to do similar for the other commands' descriptions.

Reply:
ok